Gemfile

```diff
@@ -1,9 +1,11 @@
 source 'https://rubygems.org'
 
+
 ruby '2.1.5'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.0'
 # Use mysql as the database for Active Record
```

Comment:
Can you take out pagination for now ?
We will use pagination in web. lets think of api later